src/status.js

```diff
@@ -53,6 +51,53 @@ function addList() {
   form.reset();
 }
 
+function editList(ev) {
+  const buttonId = ev.target.id;
+  const dataGet = localStorage.getItem('todoObject');
+  const data = JSON.parse(dataGet);
+  if (data) {
+    collection = data;
+  }
+  const rtest = collection[collection.findIndex((x) => x.id === parseInt(buttonId, 10))];
+  const last = collection.indexOf(rtest);
+  const test = ev.target.parentNode;
+  const editInput = test.querySelector('p');
+  editInput.contentEditable = true;
+  editInput.classList.add('test');
+  const a = test.querySelector('.fa-ellipsis-v');
+  a.style.display = 'none';
+  const b = test.querySelector('.fa-trash-o');
+  b.style.display = 'block';
+  collection[last].title = editInput.innerHTML;
+
+  editInput.addEventListener('keyup', () => {
+    collection[last].title = editInput.innerHTML;
+    localStorage.setItem('todoObject', JSON.stringify(collection));
+  });
+}
+
+function removeTodo(ev) {
+  const buttonId = ev.target.id;
+  collection = collection.filter(
+    (y) => y !== collection[collection.findIndex(
+      (x) => x.id === parseInt(buttonId, 10),
+    )],
+  );
+  localStorage.setItem('todoObject', JSON.stringify(collection));
+  ShowList(collection);
+}
+
+function removeCompleted() {
+  for (let i = 0; i < collection.length; i += 1) {
+    if (collection[i].complete === true) {
+      collection.splice(i, 1);
+      i -= 1;
+    }
+  }
+  localStorage.setItem('todoObject', JSON.stringify(collection));
+  ShowList(collection);
+}
```

Comment:
Good job adding this function but we want to make use of `Array.filter()` method to achieve this.
![image](https://user-images.githubusercontent.com/31897434/127466078-552251a8-06e5-4229-9dbd-8e4cf3c40dcb.png)